### PR TITLE
Design Cleanup

### PR DESCRIPTION
**Pull request by mtrythall**
This does a first pass of design clean up. It does the following:

1. Removes unnecessary CSS code
2. Adds appropriate logo files
3. Simplifies some of the layout code
4. Adds the social nav and footer
5. Slims down the aesthetic a little
6. Introduces missing CSS files, like those for reset

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,110 +43,161 @@
...
-              <h1 class="json-header">
-                JSON<span class="two-text">{2}</span>SDL
-              </h1>
-              <div class="starter-buttons">
-                <button
-                  id="form-button"
-                  @click.self="handleClick()"
-                  class="color7 start-button"
-                >
-                  Start with an example
```
Comment (SamuelGHill) [inline index.html:94]: Change this

```
<button
              id="form-button"
              @click.self="handleClick()"
              class="button color11"
            >
              <span>Start with an example</span>
            </button>
```

to this and the generator for start with an example will work. The convert button still works.
```
<button
              id="form-button"
              class="button color11"
            >
              <span
                @click.self="handleClick()"
              >Start with an example
              </span>
            </button>
```

Comment (SamuelGHill) [reply index.html:94]: I poorly added tabs to that. Please clean up when copying and pasting

File: index.html [hunk 2]
```diff
@@ -43,110 +43,161 @@
...
+              id="form-button"
+              @click.self="handleClick()"
+              class="button color11"
+            >
+              <span>Start with an example</span>
+            </button>
+          </header>
+
+            <section class="codedemo">
+              <div class="codedemo-display">
```
Comment (SamuelGHill) [inline index.html:94]: The responsiveness of the code is not the same as the json2sdl right now. Maybe something got deleted that was needed to keep the responsiveness?

Example is 

1. When I go to mobile, the code does not appear in the input / json side.
2. When the code is generated on desktop, the div surrounding the code editor enlarges beyond the size of the code, and does not properly wrap it. Click the "example" button and scroll the example json and you will see. I could not pin point what was deleted that made this work.

**Review by SamuelGHill — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (mtrythall) [reply index.html:94]: Makes sense. No sense in worrying about bubbling it up.

File: index.html (hunk 2 above)
Comment (mtrythall) [reply index.html:94]: @SamuelGHill This ought to be better, but not perfect. Ace Editor is doing some sizing that I think was resulting in double scrolling before (and now). The second issue is still sort of there, but not that bad, and not such that I think this should block from going up. Give it a look and LMK what you think.

**Review by SamuelGHill — APPROVED** [review]
LGTM

**Pull request merged**